content/en/guides/models/app/settings-page/user-settings.md

```diff
@@ -33,16 +33,35 @@ To add a follow badge for the `@weights_biases` account on X, you could add a Ma
 ```
 In an `<img>` tag, you can specify `width`, `height`, or both. If you specify only one of them, the image's proportions are maintained.
 
-## Teams
+## Default team
+If you are a member of more than one team, the **Default team** section allows you to configure the default team to use when a run or a Weave trace does not specify a team. If you are a member of only one team, that team is the default and this section does not appear.
+
+Select a tab to continue.
+
+{{< tabpane >}}
+{{% tab header="Multi-tenant Cloud" %}}
+Next to **Default location to create new projects in**, click thew drop-down, then select your default team.
+{{% /tab %}}
+{{% tab header="Dedicated Cloud / Self-Managed" %}}
+1. Next to **Default location to create new projects in**, click thew drop-down, then select your default team or your personal entity.
```

Comment:
These list numbers rendered fine in preview (and the same formatting is used below, too), so it's probably fine, but is there a reason the steps are all labels as `1`?

Reply:
Auto-numbering. :) This way, if I add or remove a step, I don't need to re-number all the others. This is fairly standard with Markdown processors. We don't have a hard and fast rule. I believe that @ngrayluna tends to prefer explicit numbering.